sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.aest.getApplicationAssessmentStatusDetails.e2e-spec.ts

```diff
@@ -0,0 +1,71 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import { ApplicationStatus, StudentAssessmentStatus } from "@sims/sims-db";
+
+describe("ApplicationAESTController(e2e)-getApplicationAssessmentStatusDetails", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should get application and assessment status details when requested.", async () => {
```

Comment:
can we change it to "Should get assessment status for an application when requested for an application id" ??